openedx/tests/xblock_integration/xblock_testcase.py

```diff
@@ -28,18 +28,20 @@
 
 Our next steps would be to:
 * Finish this framework
-* Move tests into the XBlocks themselves
```

Comment:
For my own edification, why remove the point about moving the tests into the XBlocks themselves?